en/components/general/wpf_to_angular_guide/create_ui_with_components.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Create your UI with Angular components
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular, UserControl, Component
+---
+
+## Create your UI with Angular components
```

Comment:
We have a couple of sentences for decorator & selector - I don't think these have to be separated anyhow. It would break the normal flow of the topic.